packages/flutter/lib/src/widgets/scroll_physics.dart

```diff
@@ -113,7 +111,7 @@ class ScrollPhysics {
   ///
   ///  * [buildParent], a utility method that's often used to define [applyTo]
   ///    methods for ScrollPhysics subclasses.
-  ScrollPhysics applyTo(ScrollPhysics ancestor) {
+  ScrollPhysics applyTo(ScrollPhysics? ancestor) {
```

Comment:
ancestor should be non-null

Reply:
This change leads to several failure in tests (that I'm not sure how to fix).

Reply:
k, i'll look into it